utils/uiTypes/proposalCreationTypes.ts

```diff
@@ -615,3 +615,8 @@ export interface DualFinanceVoteDepositForm {
   realm: string | undefined
   delegateToken: AssetAccount | undefined
 }
+
+export interface MultiChoiceProposalForm {
```

Comment:
this type just gets used once, just put it inline. 

`useState<{ governedAccount: AssetAccount | undefined
  options: string[]
}>` or whatever